test/documentation/bio.css

```diff
@@ -0,0 +1,376 @@
+body {
```

Comment:
can we just use the file from the SeqAn3 submodule?

Reply:
yes this one we can copy :+1: